### ISSUE TITLE
Version in crates.io is 0.11.0, not the latest 0.12.0

### ISSUE THREAD

**Issue by jmcastagnetto** [guoxbin/dtool#31]
Installing using `cargo` will get 0.11.0, so some options like `ts` that are in 0.12.0 are missing

```
$ dtool ts
error: The subcommand 'ts' wasn't recognized
	Did you mean 'ts2d'?
$ dtool --version
dtool 0.11.0
```

**Comment by guoxbin** [comment]
Have published 0.12.0 on crates.io

**Comment by guoxbin** [comment]
Have published 0.12.0 on crates.io
